### PR TITLE
Allow multiple paths or files as argument

### PR DESCRIPTION
**Pull request by stmh**
This MR will allow multiple file arguments, we need this functionality so we can use twigcs with [pre-commit](https://pre-commit.com/) as pre-commit passes all staged files via arguments to the linter.

### CODE REVIEW

File: src/Console/LintCommand.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ public function configure()
     {
         $this
             ->setName('lint')
-            ->addArgument('path', null, 'The path to scan for twig files.', '.')
+            ->addArgument('paths', InputArgument::IS_ARRAY | InputArgument::OPTIONAL, 'The path to scan for twig files.', ['.'])
```
Comment (Ciloe) [inline src/Console/LintCommand.php:20]: Hi, this is a breaking change fonctionnality. You must allow single path and folder without array. I think you can add tests two and update the readme.

Thank you for your contribution

**Review by Ciloe — CHANGES_REQUESTED** [review]

File: src/Console/LintCommand.php (hunk 1 above)
Comment (stmh) [reply src/Console/LintCommand.php:20]: Hi,

My change does not break existing functionality, you can still call twigcs with a single path argument.

If wanted I can add a unit-test.

Comment (Ciloe) [reply src/Console/LintCommand.php:20]: Ok, I just verified, it's ok for me ! Nice job !

**Review by Ciloe — APPROVED** [review]

**Review by OwlyCode — APPROVED** [review]
Nice addition! If you want to add a test on that part you're welcome : we have none on the command part of twigcs.
